app-admin/terraform/metadata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
gentoo-proxy-maintainers should be added last

Reply:
So my name goes first? or add the group?

Reply:
I added the proxy-maintainers project